log/consul_test.go

```diff
@@ -52,6 +52,36 @@ func TestConsul(t *testing.T) {
 		buf.Reset()
 	})
 
+	t.Run("Set level - NOOP", func(t *testing.T) {
+		logger.SetLevel(0)
+		t.Log("Set level - NOOP")
+		buf.Reset()
+	})
+
+	t.Run("Log", func(t *testing.T) {
+		logger.Log(hclog.NoLevel, "123")
+		assert.Empty(t, buf.String())
+		buf.Reset()
+		logger.Log(hclog.Trace, "123")
+		assert.Contains(t, buf.String(), "DEBUG: 123")
```

Comment:
Should be `"TRACE: 123"` instead.

Reply:
This is the consul logger and funny enough trace reports `DEBUG`. The test is passing.